faros_event.sh

```diff
@@ -329,13 +343,168 @@ function processEventTypes() {
     fi
 }
 
+function unix_seconds_to_iso8601() {
+    echo $(jq -r 'todate' <<< $1)
```

Comment:
nice! Would it be possible to do this for [this line](https://github.com/faros-ai/faros-events-cli/pull/46/files#diff-9b0b1096c617d8a0da992c12061d684a20d648e5e6b003060f9fc85c3cd549b7R826) as well? just so all dependencies on `date` are removed?

Reply:
Done :)